packages/react-conformance/package.json

```diff
@@ -25,14 +25,18 @@
     "@types/react": "16.8.25",
     "@types/react-dom": "16.8.4",
     "@types/react-test-renderer": "^16.0.0",
+    "@types/react-is": "^16.7.1",
     "@uifabric/build": "^7.0.0",
     "@uifabric/tslint-rules": "^7.2.1",
     "enzyme": "~3.10.0",
     "enzyme-adapter-react-16": "^1.15.0",
     "jest": "~24.9.0",
+    "lodash": "^4.17.15",
```

Comment:
lodash, react-docgen-typescript, and react-is should be dependencies (not devDependencies), because a consumer will need them when using this package.